cmd/chunks-inspect/main.go

```diff
@@ -17,28 +21,22 @@ var timezone = time.UTC
 func main() {
 	blocks := flag.Bool("b", false, "print block details")
 	lines := flag.Bool("l", false, "print log lines")
-	storeBlocks := flag.Bool("s", false, "store blocks, using input filename, and appending block index to it")
```

Comment:
was this command option removed intentionally? if it's just a rename of `storeBlocks` -> `export` we should keep it as `storeBlocks` since we're explicitly creating loki blocks, but in reality it looks like we've removed the export to block functionality but added block -> plaintext file support.

Reply:
It was intentionally, because the raw block data is not available by the `Block` interface, which is returned by the `Chunk` interface implementation of [Blocks](https://github.com/grafana/loki/blob/main/pkg/chunkenc/memchunk.go#L1088) for `MemChunk`.

The Block interface:
https://github.com/grafana/loki/blob/88c671162f70e075f6aa43599aa560fe7b4b5627/pkg/chunkenc/interface.go#L154-L168

The export of the raw block data was removed. But as of writing, it could be possible to implement this by using the offset information from the `Block` interface. WDYT?

I added the export for the original ingested log data which might by helpful in case of recovery.

Reply:
We should keep the existing functionality unless it's really not possible to. Even if it means keeping some duplicated code, lets do so.

Reply:
I will give it a try.